src/collectors/portcheck/portcheck.py

```diff
@@ -14,7 +14,7 @@
 [port]
 [[echo]]
 number = 8080
-
+proto = tcp
```

Comment:
I would spell out `protocol`. Three additional letters won't hurt 😉

Reply:
Done!

Reply:
Variable references now renamed to be consistent with their cfg source definition. ;)